src/PeNet/PeFile.cs

```diff
@@ -478,25 +478,58 @@ public bool HasValidCertChain(bool useOnlineCrl)
         ///     and collect the corresponding bytes in an array.
         /// </summary>
         /// <returns>A byte array with Icons, an empty array if no Icons are included.</returns>
-        public IEnumerable<byte[]> Icons()
+        public IEnumerable<byte[]> IconsAsRaw()
```

Comment:
Do we actually want to be able to export raw data without headers? Is there any value in doing so?